demo/README.md

```diff
@@ -0,0 +1,38 @@
+This directory contains the helm chart(s) used for the dhdemo application.
+
+For now, this will be a localhost minikube deployment only.  
+Once we are happy with this setup, we'll move on to a production setup.
+
+If you are a Deephaven developer / contributor,  
+make sure you have setup minikube on your local machine  
+Ubuntu users: https://phoenixnap.com/kb/install-minikube-on-ubuntu
+
+
+
+Notes:
+
+Getting locally built images into minikube:
+
+./gradlew prepareCompose
+docker-compose push
```

Comment:
Are these run from the `deephaven-core` directory?  Docs are not clear.

Reply:
mark up as code blocks:  https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks